src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -542,6 +542,7 @@ xclLoadAxlf(const axlf *buffer)
       .za_flags = flags,
       .za_ksize = 0,
       .za_kernels = NULL,
+      .za_slot_id = 0,
```

Comment:
We are not suppose to put a "slot id" here. Slot ID is something inside driver. Not from user space. @saifuddin-xilinx maybe you already aware of this. Just put a comment here to make sure we remember to clean this up.